projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -855,6 +883,30 @@ function wpcom_launchpad_get_newsletter_subscriber_count() {
 	return (int) $total_subscribers;
 }
 
+/**
+ * Determines if Stripe has been connected.
+ *
+ * @return bool Whether or not Stripe account is connected.
+ */
+function wpcom_is_stripe_connected() {
```

Comment:
should we add it to the `lib` instead? sounds useful

Reply:
Agreed -- this should be `wpcom_launchpad_is_stripe_connected()` if we're keeping it in the Launchpad code, or it should move somewhere else (where "somewhere else" could still be in jetpack-mu-wpcom).
```suggestion
function wpcom_launchpad_is_stripe_connected() {
```

Reply:
Sure. Can handle as a follow up. Your comment made me think I should double check it doesn't already exists in the lib. As far as I can tell, it doesn't, which is surprising.